### PR TITLE
adding note about project creation

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

File: docs/logistics/projects.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Creating a Long-Term OSPool Project
+
+To continue using your OSG account after the school, you will
```
Comment (osg-cat) [inline docs/logistics/projects.md:3]: “OSG Connect account”, right?

Comment (rachellombardi) [reply docs/logistics/projects.md]: Miron wants to move away from "Connect"

File: docs/logistics/projects.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+		PI Name: (e.g. faculty advisor/sponsor, director, etc.)
+		PI Email: 
+		PI Department or Organization: 
+		PI Institution: 
+		Project Field of Science: (out of 
+		https://osp.unm.edu/pi-resources/nsf-research-classifications.html)
+		Brief description of the relevant research: (1-3 sentences, perhaps linking to a webpage)  
+		
+1. Email the above to support@opensciencegrid.org, **cc'ing the PI of your research group** and
```
Comment (rachellombardi) [inline docs/logistics/projects.md:17]:
```suggestion
1. Email the above to support@osg-htc.org, **cc'ing the PI of your research group** and 
```

File: docs/logistics/projects.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+		Brief description of the relevant research: (1-3 sentences, perhaps linking to a webpage)  
+		
+1. Email the above to support@opensciencegrid.org, **cc'ing the PI of your research group** and 
+requesting that the PI confirm your membership in their group. If you are the PI, that should be indicated in 
+the email. 
+
+Once the project PI has confirmed your membership, OSG Facilitators will create a new 
+project of the form `Institution_PIName` within 2 business days. You will receive
+a notification when you are added to the new project, and a follow up email from 
+the OSG faciliation team.
```
Comment (rachellombardi) [inline docs/logistics/projects.md:24]:
```suggestion
the OSG faciliation team with instructions for how to switch from the `OSGUserSchool2022` project to your new project. 
```

**Review by rachellombardi — APPROVED** [review]

**Comment by rachellombardi** [comment]
looks good to me

**Pull request merged**